### PR TITLE
feat: Add basic content script to templates

### LINKED ISSUE #494: Add `content` to the template `entrypoints`
### Feature Request

After initializing a project through the provided command, there is no `content.ts` in the `entrypoints`. 

I would like to add `content.ts` to all templates, and if you agree to this proposal, I can create a PR.

#### Is your feature request related to a bug?

N/A

### What are the alternatives?

No.

### Additional context

No.

### PR DESCRIPTION
**Pull request by btea**
close #494

### CODE REVIEW

File: templates/react/entrypoints/content.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export default defineContentScript({
+  matches: ['*://*.google.com/*'],
+  runAt: 'document_start',
```
Comment (aklinker1) [inline templates/react/entrypoints/content.ts:3]: Let's exclude `runAt` and just use the default. Otherwise everything else looks good.

**Review by aklinker1 — CHANGES_REQUESTED** [review]

File: templates/react/entrypoints/content.ts (hunk 1 above)
Comment (btea) [reply templates/react/entrypoints/content.ts]: Done.

**Review by aklinker1 — APPROVED** [review]
Thanks!

**Review by aklinker1 — CHANGES_REQUESTED** [review]
Checks failed, you misspelled `console`

**Review by aklinker1 — APPROVED** [review]
Fixed it myself

**Comment by btea** [comment]
@aklinker1 Thanks for the fix, I made a stupid mistake. 🤣

**Pull request merged**